paymentmethod/MonduPayment.php

```diff
@@ -73,7 +64,8 @@ private function confirmOrder($order)
         $this->afterApiRequest($order);
     }
 
-    private function afterApiRequest($order) {
+    private function afterApiRequest($order)
```

Comment:
We can also handle pending state here, and set the order status to something other than paid for that case.

Reply:
Let me know what do you think about point that I described higher, thank you